.drone.star

```diff
@@ -1099,6 +1099,7 @@ def e2eTests(ctx):
             "HEADLESS": "true",
             "OCIS": "true",
             "RETRY": "1",
+            "REPORT_TRACING": "true",
```

Comment:
tracing is an expensive task which consumes some extra time, is there a reason why traces are recorded by default?

If you have a look here https://github.com/owncloud/web/blob/af989fc505ddd5a0ee283ed19f8f23d1cd07410f/tests/e2e/cucumber/environment/index.ts#L97 the e2e tests automatically enable har & trace recording in case of retry.

I would love to have a option like the `[full-ci]` flag to enable it but not as a default.

Reply:
But if developer starts CI without flags, he doesn't get a chance to see the cause of the error. He has to restart CI with a flag. I thought the default trace, video and har, as an option with the flag

Reply:
I think you're right, it should be an option really.